iotqatools/mongo_utils.py

```diff
@@ -209,6 +209,79 @@ def drop_database(self):
         except Exception, e:
             assert False, " ERROR - Deleting a database %s in MongoDB...\n %s" % (self.current_collection, str(e))
 
+    def get_all_databases(self):
+        """
+        Get all databases in mongo
+        :return: Database array found
+        """
+        arr_dbs = self.client.database_names()
+        return arr_dbs
+
+    def get_all_databases_with_prefix(self, l_prefix):
+        """
+        Get all databases that starts with the prefix
+        :param l_prefix: Database prefix expected
+        :return: array with databases found
+        """
+        l_dbs = []
+        arr_dbs = self.client.database_names()
```

Comment:
I recommend to call the `get_all_databases` method because it is protected with exceptions